tf_encrypted/keras/layers/dense_test.py

```diff
@@ -37,6 +37,9 @@ def _core_dense(self, **layer_kwargs):
     agreement_test(tfe.keras.layers.Dense,
                    kwargs=kwargs,
                    input_shape=input_shape)
+    layer_test(tfe.keras.layers.Dense,
```

Comment:
Do we want this test only for `Dense`, `Conv2D` and `Activation` or for all the layers to be consistent?

Reply:
probably all the layers